backend/hexa/workspaces/utils.py

```diff
@@ -107,8 +108,14 @@ class DHIS2MetadataQueryType(StrEnum):
     INDICATOR_GROUPS = "INDICATOR_GROUPS"
 
 
+class IASOMetadataQueryType(StrEnum):
```

Comment:
To keep it consistent we could name everything:
```suggestion
    IASO_FORMS = "IASO_FORMS"
    IASO_ORG_UNITS = "IASO_ORG_UNITS"
    IASO_PROJECTS = "IASO_PROJECTS"```